app/decorators/service_provider_session_decorator.rb

```diff
@@ -4,6 +4,17 @@ class ServiceProviderSessionDecorator
 
   DEFAULT_LOGO = 'generic.svg'.freeze
 
+  SP_ALERTS = {
+    'CBP Trusted Traveler Programs' => {
+      i18n_name: 'trusted_traveler',
+      learn_more: 'https://login.gov/help/trusted-traveler-programs/sign-in-doesnt-work/',
+    },
+    'USAJOBS' => {
+      i18n_name: 'usa_jobs',
+      learn_more: '/',
```

Comment:
Are we planning on creating a new help page for USA Jobs? Is that why there is no URL configured here? If so, we should wait until that page is ready before merging this, right?

Reply:
We should have a new faq article for USA jobs before launch of usa jobs. Just pinged @jkim1217 about that.

Reply:
OK, so I propose we either wait to merge this PR until that FAQ page is ready, which is fine since this PR doesn't provide any benefit to current users. Or, we can merge this, but open a new issue to update the learn more link once it's ready, and tag the issue as must be resolved before launch.